project/Dependencies.scala

```diff
@@ -4,7 +4,7 @@ import Keys._
 object Dependencies {
 
   val AkkaVersion = "2.5.14"
-  val AkkaHttpVersion = "10.0.13" // TODO update to 10.1.x as soon as possible
+  val AkkaHttpVersion = "10.1.3"
```

Comment:
This means that we will not support using Akka Management with applications that are still using Akka HTTP 10.0.x. I think that is fine since it's time to more aggressively phase out 10.0 anyway, but wanted to highlight that so it doesn't come as surprise after the fact for anyone.

Reply:
Keep in mind that the current stable versions of Play and Lagom still use 10.0. A lot of Lagom users especially will have this dependency brought in by Lightbend Orchestration or Cluster Bootstrap.

I'm not sure if it would cause problems to evict 10.0 in these situations (I don't think we've seen any evidence that it would) but it won't be tested thoroughly in this combination, and users might not even be aware that it's happening.

Reply:
Then we should stay on 10.0.13 and try to work around then bincompat issue